Scalability Metadata Placement

eleft opened the pull request:
Proposed changes on the location of scalability metadata in a coded video sequence, as well as resolution of possible duplicate scalability metadata in a coded video sequence. This update replaces PR209.

eleft committed:
Update 07.bitstream.semantics.md

Proposed changes on the location of scalability metadata in a coded video sequence, as well as resolution of possible duplicate scalability metadata in a coded video sequence. This update replaces PR209.

peterderivaz commented:
@eleft On the call we were discussing having a note to explain that not all layers may be present, for example, due to intermediate processing entities.  Do you think this is still helpful?  (At the moment it seems ambiguous as to whether the scalability metadata OBUs need to be rewritten or not.)

peterderivaz commented on 07.bitstream.semantics.md:
```diff
@@ -603,6 +603,13 @@ entity identified by the ITU-T T.35 country code and terminal provider code.
+coded video bitstream without having to decode individual frames. 
+{:.alert .alert-info }
+
+Scalability metadata should be placed between the first sequence header and the first frame header of a coded video sequence. The information present in a scalability metadata OBU applies to the entire coded video sequence it is contained. Redundant copies of a scalability metadata OBU may occur in any temporal unit of a coded video sequence.
```
Perhaps "it is contained." should be "it is contained in."?

This reads a bit like all bitstreams should contain scalability metadata.  Perhaps start the sentence with "If scalability metadata is present it should be ..."

eleft replied:
Thanks Peter. All valid points. Stephen repeated the first one, and added one more. I will produce a revision that integrates them.

eleft committed:
Reference to maximal structure, optional presence, and further scope refinement.

Incorporated comments from Stephen and Peter.

eleft committed:
Minor edits for clarity

Per Stephen's suggestion, I changed “original coded video bitstream” to "original coded video sequence” in the note. Also, “and only that sequence” instead of “and only that” further down.
